components/esp_websocket_client/include/esp_websocket_client.h

```diff
@@ -88,7 +88,7 @@ typedef struct {
     char                        *headers;                   /*!< Websocket additional headers */
     int                         pingpong_timeout_sec;       /*!< Period before connection is aborted due to no PONGs received */
     bool                        disable_pingpong_discon;    /*!< Disable auto-disconnect due to no PONG received within pingpong_timeout_sec */
-
+    int                         ping_interval_sec;          /*!< Websocket ping interval */
```

Comment:
Why is it given is seconds here but stored in milliseconds above? I am new to this IDF so I do not know if this may be the default among other components too...

Reply:
- Because `pingpong_timeout_sec` was already there. And 
I think I'd never to make detailed settings on the order of milliseconds.
- In esp_websocket_client.c, comparing with _tick_get_ms(), it is milliseconds